Websites/perf.webkit.org/public/include/uploaded-file-helpers.php

```diff
@@ -194,4 +194,4 @@ function prune_old_files($db, $size_needed, $remote_user)
     return FALSE;
 }
 
-?>
+?>
```

Comment:
Are you sure this line break addition is intended?

Reply:
This is actually removes `\n`
```
@@ -194,4 +194,4 @@ function prune_old_files($db, $size_needed, $remote_user)
     return FALSE;
 }
 
-?>
+?>
\ No newline at end of file
```

Reply:
Will have a separate change to use open php tags anyway.